amundsen_application/static/js/utils/numberUtils.ts

```diff
@@ -12,3 +12,9 @@ export function formatNumber(value) {
   }
   return Intl.NumberFormat().format(value);
 }
+
+export function isNumber(value) {
+  if (typeof value == 'string') {
```

Comment:
We can type value as a string, and just have `return !Number.isNaN(Number(value));` We wouldn't need the check because code won't build if someone uses it incorrectly and passes something that isn't a string, and we would also want this method to always return something.

As is, someone can get `isNumber(100)` would return undefined

nit: we can also type the return type as a boolean.  `isNumber(value: string): boolean`